packages/pg/lib/connection-parameters.js

```diff
@@ -104,6 +105,13 @@ class ConnectionParameters {
         throw new Error(' You need to enable Load Balance feature to use Topology Aware! ')
       }
     }
+    this.ybServersRefreshInterval = Number(this.ybServersRefreshInterval)
+    if(isNaN(this.ybServersRefreshInterval) || !Number.isInteger(this.ybServersRefreshInterval)){
+      throw new Error(' You need to Enter valid Refresh Interval ')
+    }
+    if(this.ybServersRefreshInterval<0 || this.ybServersRefreshInterval>600){
+      this.ybServersRefreshInterval = 300
```

Comment:
We are silently changing the refresh interval if it is >600, probably we must throw error inthis case as well so that user get to know that refresh interval can't be set more than 600?

Reply:
Discussed, handled in Docs.

Reply:
Yes, it'll be good to have a section for YBDB-addtions in the README and talk about smart features (or at least point reader to the page in YugabyteDB docs)